simexpal/base.py

```diff
@@ -1117,6 +1137,17 @@ def __init__(self, cfg, experiment, instance, repetition):
 	def config(self):
 		return self._cfg
 
+	@property
+	def internal_name(self):
+		internal_name = self.experiment.name
+		if self.experiment.variation:
+			internal_name += ' ~ ' + ', '.join([variant.name for variant in self.experiment.variation])
```

Comment:
Drop the spaces in `internal_name` (also for the revision).

Reply:
Done